File: frameworks/Java/fair/java-kora2-jdbc-repository/src/main/java/io/techempower/benchmark/kora/model/Fortune.java
package io.techempower.benchmark.kora.model;

import io.koraframework.database.common.annotation.Column;
import io.koraframework.database.common.annotation.Id;
import io.koraframework.database.jdbc.annotation.EntityJdbc;

@EntityJdbc
public record Fortune(@Id @Column("id") int id,
                      @Column("message") String message) {
}

File: frameworks/Java/fair/kotlin-kora2-jdbc-repository/src/main/kotlin/io/techempower/benchmark/kora/model/Fortune.kt
package io.techempower.benchmark.kora.model

import io.koraframework.database.common.annotation.Column
import io.koraframework.database.common.annotation.Id
import io.koraframework.database.jdbc.annotation.EntityJdbc

@EntityJdbc
data class Fortune(
    @field:Id @field:Column("id") val id: Int,
    @field:Column("message") val message: String,
)
